bridge/src/main/java/io/stargate/bridge/service/SchemaHandler.java

```diff
@@ -174,11 +176,25 @@ private static CqlTable buildCqlTable(Table table) throws StatusException {
   }
 
   private static CqlIndex buildSecondaryIndex(SecondaryIndex index) {
-    return CqlIndex.newBuilder()
-        .setName(index.name())
-        .setColumnName(index.column().name())
-        .setCustomType(StringValue.newBuilder().setValue(index.indexTypeName()).build())
```

Comment:
`index.indexTypeName()` is hard-coded to "Secondary index". That's not very useful in gRPC since the client already knows that it's dealing with a `CqlIndex` (as opposed to a materialized view or table).
I removed the `customType` field completely.